ee-feature-pack/layer-metadata-tests/pom.xml

```diff
@@ -26,6 +26,11 @@
     <description>Tests for the layer metadata rules</description>
     <packaging>jar</packaging>
 
+    <properties>
+        <!-- Don't try to deploy the module -->
```

Comment:
Just an opinion, but I find this comment redundant, since this is well known what this property does (11x).

https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html#skip

Reply:
it is already present in our testsuite/pom.xml